GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/LevelerViewModel.cs

```diff
@@ -9,8 +9,12 @@ public class LevelerViewModel : ViewModelBase
         public TableUser User { get; set; }
         public ICommand ToggleLeveler { get; private set; }
         const string MAX_LEVEL = "Five";
+        const int DEFAULT_CLASSIFICATIONS_UNTIL_UPGRADE = 5;
+        const int DEFAULT_CLASSIFICATIONS_COUNT = 5;
+        const string DEFAULT_CLASSIFICATION_LEVEL = "One";
+        const bool DEFAULT_CLASSIFICATION_OPEN = false;
```

Comment:
Should these all be `private`?

Reply:
They're implicitly `private`, but I can make it explicit.